tests/exports-in-file.test.ts

```diff
@@ -0,0 +1,33 @@
+import assert from 'node:assert/strict';
+import test from 'node:test';
+import { main } from '../src/index.js';
+import { resolve } from '../src/util/path.js';
+import baseArguments from './helpers/baseArguments.js';
+import baseCounters from './helpers/baseCounters.js';
+
+const cwd = resolve('tests/fixtures/exports-in-file');
+
+test('Find unused exports respecting ignoreExportsUsedInFile', async () => {
+  const { issues, counters } = await main({
+    ...baseArguments,
+    cwd,
+  });
```

Comment:
Shouldn't this include the `ignoreExportsUsedInFile: true` option?

Reply:
Ah, the tests weren't testing the way I thought they were. Sorry about that. Should be fixed now.